Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -1611,4 +1615,34 @@ BOOST_PYTHON_MODULE(rdMolDescriptors) {
               docString.c_str());
 
 #endif
+  
+#ifdef RDK_HAS_EIGEN3
+  python::scope().attr("_BCUT2D_version") =
+      RDKit::Descriptors::BCUT2DVersion;
+  std::vector<double> (*BCUT)(const RDKit::ROMol&) = & RDKit::Descriptors::BCUT2D;
+  std::pair<double,double> (*BCUT_props)(const RDKit::ROMol&, const std::vector<double>&) = & RDKit::Descriptors::BCUT2D;
+  std::pair<double,double> (*BCUT_atomprops)(const RDKit::ROMol&, const std::string&) = & RDKit::Descriptors::BCUT2D;
+  docString = "Returns the standard 2D BCUT2D descriptors vector";
+    
+  python::def("BCUT2D", BCUT,
+              (python::arg("mol")),
+              docString.c_str());
+
+  python::class_<std::pair<double, double> >("BCUTPair")
+    .def_readwrite("first", &std::pair<double, double>::first)
+    .def_readwrite("second", &std::pair<double, double>::second)
+    .def_readwrite("higest", &std::pair<double, double>::first)
```

Comment:
I don't think including `highest` (which is mis-spelled here) and `lowest` add anything